domains/authorization/Authorization.API/API/Migrations/20240730083845_AddUniqueTermsVersionConstraint.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace API.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddUniqueTermsVersionConstraint : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateIndex(
+                name: "IX_Terms_Version",
```

Comment:
Why index on version?

Reply:
In order to enforce the unique constraint, it needs to look it up in the database, and creating an index for that, makes it more performant.

Reply:
But i dont believe you need to make it an index to get the unique constraint? I dont think we get that many versions that you will notice any performance issues